src/components/expression/expression-renderer.js

```diff
@@ -178,6 +178,42 @@ genotet.ExpressionRenderer.Profile = function(params) {
   this.color = params.color != null ? params.color : null;
 };
 
+/**
+ * Zoom status object storing the status of expression matrix.
+ * @param {!{
+ *   matrixName: (?string|undefined),
+ *   geneRegex: (?string|undefined),
+ *   conditionRegex: (?string|undefined),
+ *   geneNames: (?string|undefined),
+ *   conditionNames: (?string|undefined)
+ * }} params
```

Comment:
Everything can be undefined is a bit too permissive. Please be as strict as possible. It seems that matrixName is never undefined in your usage, is it?